### PR TITLE
Credit card

### PR DESCRIPTION
**Pull request by mosch**
It's a credit card input. We don't need it right now, but I don't want to throw away.
[See how cool it is.](https://deploy-preview-299--allthings-elements.netlify.com/?selectedKind=CreditCardInput&selectedStory=with%20credit%20card&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel)

Checklist:

- [x] Test added / Snapshots updated
- [x] Documentation added / updated

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
     "prop-types": "15.6.2",
     "react-broadcast": "0.7.1",
     "react-media": "1.8.0",
-    "react-motion": "0.5.2"
+    "react-motion": "0.5.2",
```
Comment (dan-lee) [inline package.json]: Don't think we need `react-motion`, right?

File: doc/reference/molecules/CreditCardInput.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!-- 
+This is an auto-generated markdown. 
+You can change it in "src/molecules/CreditCardInput.jsx" and run build:docs to update this file.
+-->
+# CreditCardInput
+CreditCardInput is used to enter credit card details.
+
+```example
+<PhoneInput name="phone" placeholder="Example Placeholder" defaultValue="4907615555555" required />
```
Comment (dan-lee) [inline doc/reference/molecules/CreditCardInput.md]: Maybe we should put `CreditCardInput` as an example here 😇

**Comment by dan-lee** [comment]
Really cool ✌️

File: src/molecules/CreditCardInput.jsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+class CreditCardInput extends Component {
+  static propTypes = {
+    /** Indicates that this field is required */
+    required: bool,
+    /** Called, when the users changes something */
+    onChange: func,
+  }
+
+  renderNumber = (ref, maskedProps) => (
+    <Input
```
Comment (fivenp) [inline src/molecules/CreditCardInput.jsx]: love this component - but please lets trigger the numpad keyboard on focus :)

Comment (mosch) [reply src/molecules/CreditCardInput.jsx]: Unfortunately this is not possible. We use underlying features only available in non nuerm fields.

**Comment by fivenp** [comment]
?

**Pull request closed without merging**